apiary-receiver/apiary-receiver-sqs/src/main/java/com/expedia/apiary/extensions/receiver/sqs/messaging/SqsMessageReader.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright (C) 2018-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.apiary.extensions.receiver.sqs.messaging;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.Iterator;
+import java.util.Optional;
+
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+import com.amazonaws.services.sqs.model.DeleteMessageRequest;
+import com.amazonaws.services.sqs.model.Message;
+import com.amazonaws.services.sqs.model.ReceiveMessageRequest;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.expedia.apiary.extensions.receiver.common.messaging.JsonMetaStoreEventDeserializer;
+import com.expedia.apiary.extensions.receiver.common.messaging.MessageDeserializer;
+import com.expedia.apiary.extensions.receiver.common.messaging.MessageReader;
+import com.expedia.apiary.extensions.receiver.common.error.SerDeException;
+import com.expedia.apiary.extensions.receiver.common.event.ListenerEvent;
+
+public class SqsMessageReader implements MessageReader {
+  private static final Integer DEFAULT_POLLING_WAIT_TIME = 10;
```

Comment:
What's the unit ms? can you add that in the name?

Reply:
Yep seconds, I'll rename this to `DEFAULT_POLLING_WAIT_TIME_SECONDS`